nexus/agents/api/views.py

```diff
@@ -391,8 +391,13 @@ class ProjectCredentialsView(APIView):
     permission_classes = [IsAuthenticated, ProjectPermission]
 
     def get(self, request, project_uuid):
-        credentials = Credential.objects.filter(project__uuid=project_uuid)
-        
+        active_agents = ActiveAgent.objects.filter(team__project__uuid=project_uuid)
```

Comment:
It is necessary to first check whether the user has permission on this project. The current format opens the door for users to obtain credentials for projects that they do not have permission for.